google-cloud-datastore/acceptance/datastore/datastore_test.rb

```diff
@@ -554,6 +554,40 @@
       _(entities.count).must_equal 4
     end
 
+    it "should fetch entities filtered through a simple Filter object" do
+      datastore = Google::Cloud::Datastore.new
+      filter = datastore.filter("alive", "=", true)
+      query = datastore.query("Character")
+        .ancestor(book)
+        .where(filter)
+      entities = datastore.run query
+      _(entities.count).must_equal 4
+    end
+
+    it "should fetch entities filtered by AND operator" do
+      datastore = Google::Cloud::Datastore.new
+      filter = datastore.filter("name", "=", "Arya")
+                        .and("alive", "=", true)
+                        .and(datastore.filter("appearances", "=", 33))
+      query = datastore.query("Character")
+        .ancestor(book)
```

Comment:
Nit: Please fix the alignment of the chained methods

Reply:
Fixed the alignment.